infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/RestApiBuilder.java

```diff
@@ -129,6 +145,20 @@ public RestApi build() {
     return new RestApi(app, restApiDocs);
   }
 
+  private void checkAccessFile(final Path path) {
+    if (!path.toFile().exists()) {
+      final Bytes generated =
+          Bytes.of(RandomStringUtils.randomAlphanumeric(16).getBytes(StandardCharsets.UTF_8));
+      try {
+        LOG.info("Initializing API auth access file {}", path.toAbsolutePath());
+        Files.writeString(path, generated.toUnprefixedHexString(), StandardCharsets.UTF_8);
```

Comment:
If we're writing this as a hex string anyway, we may as well use the full scope of bytes and just use `Bytes.random(16)` to generate it.  If we're restricting to alphanumeric characters we should use 32 chars and write them directly instead of converting to hex.